### DISCUSSION TITLE
Soy nuevo

### DISCUSSION THREAD

**Discussion by gersondlarosa** [LaunchX-InnovaccionVirtual/CursoIntroPython#69] [Q&A] [answered]
Buena noche me prodrian orientar para iniciar mis catas vengo en cero

**Comment by KevinChavarin** [comment]
En cada módulo viene un archivo para lectura y práctica y después la kata, mi recomendación es que practiques, leas y por último realices la kata.
P.D: Instala Python 3.10 desde la Microsoft Store

**Reply by WilleMorales** [reply]
soy nuevo donde podemos ver las katas ?
solo veo los módulos y los ejercicios

**Reply by KevinChavarin** [reply]
> soy nuevo donde podemos ver las katas ?
> solo veo los módulos y los ejercicios 

Entras a las carpetas de los módulos están tanto los ejercicios como las katas

**Comment by DiegoHC06** [comment]
Primero tienes que leer el documento y luego vas al kata, pero antes ya revisaste la primera misión que es conocer el GitHub y las herramientas a usar.
Si es así, entonces pues ir al Teams, hay un apartado que se llama "Primer misión", en ese apartado viene el documento.  llamado "esquema de la misión", aun que igual puedes ir leyendo y luego realizas el kata.

**Reply by Texhnolyze47** [reply]
También puedes ver la retrasmisión donde explica varias cosas sobre las katas
[https://teams.microsoft.com/l/meetup-join/19%3ameeting_YzAyNzI0YmQtNzIyOS00OWRjLWI4YWMtYzc2MDVhZGFhNGM0%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%227b10324a-57d4-43ce-95fe-68f52b498664%22%2c%22IsBroadcastMeeting%22%3atrue%7d](url)

**Comment by gersondlarosa** [comment]
Muchas gracias después del life empiezo

**Reply by ArielCanedo** [reply]
Perfecto @gersondlarosa  cualquier duda puedes crear una discusión, trata de ser especifico con tus dudas para ayudarte

**Comment by doguedogue** [comment, marked as answer]
No olvides marcar como CONTESTADA tu pregunta si así lo consideras.

![image](https://user-images.githubusercontent.com/23409026/153456370-fc3f7efa-4247-406d-bcbc-2c96128c994e.png)

**Comment by ChanLuis** [comment]
De preferencia específica la pregunta en el título
